lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/migration/MigratableTableDAOImplAutowireTest.java

```diff
@@ -220,6 +220,7 @@ public void testGetPrimaryMigrationTypes(){
 		expectedPrimaryTypes.add(MigrationType.DATA_ACCESS_REQUEST);
 		expectedPrimaryTypes.add(MigrationType.DATA_ACCESS_SUBMISSION);
 		expectedPrimaryTypes.add(MigrationType.DATA_ACCESS_SUBMISSION_SUBMITTER);
+		expectedPrimaryTypes.add(MigrationType.DOWNLOAD_LIST);

```

Comment:
replace with a test that fails if you forget to register a new type. (remove test that files if you do the right thing).